src/utils/process.js

```diff
@@ -65,30 +64,41 @@ async function findProcessUnix (browserId) {
     return findProcessIdUnix(browserId, output);
 }
 
-async function checkUnixProcessIsKilled (processId) {
+async function isUnixProcessKilled (processId) {
     const output = await getProcessOutputUnix();
 
-    if (isProcessExistUnix(processId, output)) {
-        await delay(CHECK_KILLED_DELAY);
+    return !isUnixProcessExist(processId, output);
+}
 
-        await checkUnixProcessIsKilled();
-    }
+async function killUnixProcessSoft (processId) {
+    process.kill(processId);
+}
+
+async function killUnixProcessHard (processId) {
+    process.kill(processId, 'SIGKILL');
 }
 
 async function killProcessUnix (processId) {
-    let timeoutError = false;
+    let softTries = 0;
+    let unixProcessKilled = false;
 
-    process.kill(processId);
+    do {
+        await killUnixProcessSoft(processId);
+        softTries++;
+        await delay(CHECK_KILLED_DELAY);
+        unixProcessKilled = await isUnixProcessKilled(processId);
+    } while (!unixProcessKilled && softTries < 2);
```

Comment:
Please, use constants for string and number literals, except -1,0,1 and ''.

So could you please extract 'SIGKILL' and 2 to constants?